agent/consul/gateways/bind.go

```diff
@@ -21,14 +23,20 @@ type gatewayRefs = map[configentry.KindName][]structs.ResourceReference
 // The function returns a list of references to the modified BoundAPIGatewayConfigEntry objects,
 // a map of resource references to errors that occurred when they were attempted to be
 // bound to a gateway.
```

Comment:
Let's also be sure to update this comment.

Reply:
👍 prior to merge I'll be going over each of the docstrings here and adjusting a bunch of comments.

Reply:
Great, just wanted to check